src/lib/helpers/omit.ts

```diff
@@ -0,0 +1,14 @@
+export const omit =
+  <T extends object, K extends string>(keys: readonly K[]) =>
+  (obj: T): Omit<T, K> => {
```

Comment:
I decided to curry this and change the order of the arguments since that is easier to use to create partially applied functions.